### PR TITLE
added repoquery-3 test

### PR DESCRIPTION
**Pull request by glum23**
added scenarios for dnf repoquery options --all, --installed, --available, --upgrades, --recent, --extras, --repo

### CODE REVIEW

**Comment by j-mracek** [comment]
@glum23 Sorry I forget about this review

File: dnf-docker-test/features/repoquery-3.feature [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+         | Package      | Tag      | Value     |
+         | XTest        | Version  | 2         |
+         | TestA        | Version  | 2         |
+         | TestB        | Version  | 2         |
+
+  # --extras: installed pkgs, not from known repos
+  Scenario: dnf repoquery --extras (when there are such pkgs)
+       When I successfully run "rpm -i TestA-1*.rpm" in repository "base"
+        And I successfully run "dnf repoquery --extras"
+       Then the command stdout should match regexp "TestA.*1-1"
```
Comment (j-mracek) [inline dnf-docker-test/features/repoquery-3.feature]: I would recommend also negative step like ``Then the command stdout should not match regexp "TestB.*1-1"``

Comment (glum23) [inline dnf-docker-test/features/repoquery-3.feature]: done

**Comment by j-mracek** [comment]
Thanks for contribution

**Pull request merged**